dpgen/generator/arginfo.py

```diff
@@ -118,8 +142,10 @@ def model_devi_jobs_args() -> List[Argument]:
     doc_model_devi_v_trust_hi = 'Upper bound of virial for the selection. If dict, should be set for each index in sys_idx, respectively. Should be used with DeePMD-kit v2.x.'
 
     args = [
+        model_devi_jobs_template_args(), 
+        model_devi_jobs_rev_mat_args(),
```

Comment:
`sys_rev_mat` can be inserted here

https://github.com/deepmodeling/dpgen/blob/5ae6b2f70d191e20ba0794bed2ef03d9e4162f1f/dpgen/generator/run.py#L712